src/textual/css/tokenizer.py

```diff
@@ -248,11 +248,14 @@ def get_token(self, expect: Expect) -> Token:
         line = self.lines[line_no]
         match = expect.match(line, col_no)
         if match is None:
+            error_line = line[col_no:].rstrip()
```

Comment:
Slight tweak to formatting. Checks if there is a semi-colon at the end of the line, before suggesting it.